packages/openapi3/test/tsp-openapi3/output/petstore-swagger/main.tsp

```diff
@@ -54,7 +54,7 @@ model Order {
 model Customer {
   id?: int64;
   username?: string;
-  address?: Address[];
+  address?: Array<Address>;
```

Comment:
I don't think we should convert array to `Array<T>` but keep the `T[]` form

Reply:
I changed it to make it easier to handle when the element type is an enum, but can update to only use this syntax when the element is an enum.

Reply:
ah I see, can always write `("foo" | "bar")[]`

Reply:
if you always wrap in parentheses the formatter will cleanup uncessary ones

Reply:
Awesome - I was just checking if that would work.